File: CodeLibrary/TabHost/BottomNavigationView/AndroidJavaDesignBottomNavigationView/app/src/main/java/com/jackshi/androidjavatabhostfordesign/utils/MessageEvent.java
package com.jackshi.androidjavatabhostfordesign.utils;

public class MessageEvent {
    int type;
    Boolean flag;
    int position;

    public MessageEvent(int type, Boolean flag) {
        this.type = type;
        this.flag = flag;
    }

    public MessageEvent(int type, int position) {
        this.type = type;
        this.position = position;
    }

    public int getType() {
        return type;
    }

    public void setType(int type) {
        this.type = type;
    }

    public Boolean getFlag() {
        return flag;
    }

    public void setFlag(Boolean flag) {
        this.flag = flag;
    }

    public int getPosition() {
        return position;
    }

    public void setPosition(int position) {
        this.position = position;
    }
}

File: CodeLibrary/TabHost/BottomNavigationView/AndroidXkotlinSystemCreateBottomNavigationView/app/src/main/java/com/jackshi/androidx_kotlin/MessageEvent.kt
package com.jackshi.androidx_kotlin

class MessageEvent {
    var type: Int
    var flag: Boolean = false
    var position = 0

    constructor(type: Int, flag: Boolean) {
        this.type = type
        this.flag = flag
    }

    constructor(type: Int, position: Int) {
        this.type = type
        this.position = position
    }
}